code/modules/research/techweb/all_nodes.dm

```diff
@@ -224,6 +224,8 @@
 		"comp_speech",
 		"comp_speech",
 		"comp_string_contains",
+		"comp_tempsensor",
+		"comp_pressuresensor",
```

Comment:
these should be in alphabetical order, so comp_tempsensor should be around components starting with "comp_t", similar thing for comp_pressuresensor.

Reply:
comp_tempsensor is in a good spot actually but comp_pressuresensor needs to be moved